app/controllers/income_details_per_member_controller.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class IncomeDetailsPerMemberController < ManyMemberStepsController
+  private
+
+  def skip?
+    current_snap_application.
+      members.
+      none? { |member| member.employed? || member.self_employed? }
```

Comment:
> skip if no members are employed or self-employed

Is that right?

I'm asking because I'm wondering if there's any conditional that might be the opposite approach to the `none?`. For some reason unpacking `none?`, like `unless`, makes my brain bend a little bit.